rke/k8s_rke_system_images.go

```diff
@@ -3162,9 +3162,9 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 		},
 		// Enabled out of band post v2.4.8
```

Comment:
Can we add in 2.5.0 mention since this is for 2.5 branch `Enabled out of band post 2.4.8 and 2.5.0` and apply for all versions?